resolwe_bio/processes/alignment/subread.yml

```diff
@@ -56,8 +56,8 @@
           default: 1
           description: >
             Specify the number of threads/CPUs used for mapping
-        - name: unique_reads
-          label: Output uniquely mapped reads
+        - name: multiMapping
```

Comment:
Use the same naming conventions as for the rest of the inputs. Avoid camel case. Revise the description.